### PR TITLE
Add Tabs Selector to Metadata Fields

### PR DESCRIPTION
**Pull request by tanyaLibatter**

### CODE REVIEW

File: src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.scss [hunk 1]
```diff
@@ -10,5 +10,11 @@
       width: 190px;
     }
   }
+
+  ::ng-deep {
+    .ui-dropdown.ui-widget{
+      width: 176px !important;
```
Comment (dvh91) [inline src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.scss]: Isn't that 100% width of it's container?

File: src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.ts [hunk 2]
```diff
@@ -201,10 +208,12 @@ export class AddNewMetaComponent implements  OnInit, AfterViewInit, OnDestroy {
         new KalturaTranslationToken({language: 'eng', value: formValue.label})
       ],
       systemName: formValue.identifier,
-      helpText: formValue.helpText
+      helpText: formValue.helpText,
```
Comment (dvh91) [inline src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.ts]: comma without another property.

File: src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.scss (hunk 1 above)
Comment (tanyaLibatter) [reply src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.scss]: no, container width is only 96px

File: src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.ts (hunk 2 above)
Comment (tanyaLibatter) [reply src/applications/settings-metadata-templates-app/media-assets-type/add-new-meta/add-new-meta.component.ts]: fixed

**Pull request merged**